clients/src/test/java/org/apache/kafka/common/utils/MockTime.java

```diff
@@ -64,7 +64,7 @@ private void maybeSleep(long ms) {
     }
 
     @Override
-    public void sleep(long ms) {
+    public synchronized void sleep(long ms) {
```

Comment:
These changes were made when the time was a `long` rather than `AtomicLong`. They are no longer needed, removed.